### ISSUE TITLE
Platforms Supported

### ISSUE THREAD

**Issue by Deltatiger** [octalmage/robotjs#134]
This is just a discussion to get some other features working.

What are the different OSes that will be supported by RobotJS ?
Its better if this list is prepared and maintained. This will help us avoid using certain API which are deprecated or not available. For eg.) Is windows XP supported ?

**Comment by octalmage** [comment]
I'd say Windows back to XP, Mac back to Mavericks, and I'm not sure about Linux. It gets more complicated there.

**Comment by octalmage** [comment]
Should we support XP? I guess maybe not if Microsoft isn't supporting it. I just use it in a VM, because that's the build environment I have. 

Also should we support versions of Mac OSX later than Mavericks? Apple has made it pretty easy to upgrade these days (it's free), so most people should be on the latest. I say at least one version back is fair.

**Comment by octalmage** [comment]
Actually, if the Windows OS supports the version of Visual Studio suggested by node-gyp, then maybe we should support it? I'm not sure how to handle this.

**Comment by Deltatiger** [comment]
@octalmage  I was thinking something along the line of N - 2 or N - 3. So its Windows 10 that is the current generation there will be support till Windows 7 ( in terms of N - 3). Of course there will have to be some exceptions. 
But again I am with you in the matter that there is no easy way to handle this. But it has to be done.

**Comment by rezen** [comment]
I was able to install/run successfully on  Win7. I would add to the readme, to point people to this guide to ensure they can build with node-gyp. https://github.com/Microsoft/nodejs-guidelines/blob/master/windows-environment.md#environment-setup-and-configuration

**Comment by octalmage** [comment]
@rezen thanks for that! Were you able to get it working with the Standalone C++ Build Tools?

**Comment by rezen** [comment]
@octalmage yup, it works perfect after following Option 1 (Standalone C++ Build Tools) steps!

**Comment by lesmo** [comment]
So... call me crazy, but I'll be deploying this on Windows XP. I'll let you know what happens.

**Comment by octalmage** [comment]
@lesmo My build machine is Windows XP, so you should be fine! I believe I use Visual Studio 2008.

**Comment by octalmage** [comment]
So we dropped official XP support, and Node 0.8. We still need a section in the readme. 

- [ ] Add supported platforms to the readme.

**Comment by octalmage** [comment]
Closing stale issues as part of a cleanup. If this is still an issue, please reopen and I'll take a look. Thanks!
